src/app/payments/helpers.ts

```diff
@@ -1,5 +1,5 @@
 import { btcFromUsdMidPriceFn, usdFromBtcMidPriceFn } from "@app/prices"
-import { Accounts } from "@app"
+import { addNewContact } from "@app/accounts/add-new-contact"
```

Comment:
Unless this was somehow necessary I wouldn't change this. We have some wrapping on functions for opentelemetry when they're accessed like `import { Accounts } from "@app"`